src/celestia/qt/qtsolarsystembrowser.cpp

```diff
@@ -100,22 +100,65 @@ class SolarSystemTreeModel : public QAbstractTableModel, public ModelHelper
 };
 
 
+class SSTMFilterPredicate
+{
+public:
+	SSTMFilterPredicate();
+	bool operator()(const SolarSystemTreeModel::TreeItem* ti) const;
+	
+	unsigned int objectTypeMask;
```

Comment:
1. Unused variable objectTypeMask please remove it
1. As I see below it's RenderFlag, so uint64_t or better Renderer::RenderFlags should be used if you plan to implement it's usage. But Solar System Browser shouldn't use renderer flags to display objects.